vulnerabilities/improvers/vulnerability_kev.py

```diff
@@ -0,0 +1,61 @@
+import logging
+from typing import Iterable
+
+from django.db.models import QuerySet
+from sphinx.util import requests
+
+from vulnerabilities.improver import Improver
+from vulnerabilities.improver import Inference
+from vulnerabilities.models import Advisory
+from vulnerabilities.models import Alias
+from vulnerabilities.models import Kev
+
+logger = logging.getLogger(__name__)
+
+
+class VulnerabilityKevImprover(Improver):
+    """
+    Known Exploited Vulnerabilities Improver
+    """
+
+    @property
+    def interesting_advisories(self) -> QuerySet:
+        return [Advisory.objects.all()[0]]
+
+    def get_inferences(self, advisory_data) -> Iterable[Inference]:
+        """ """
+        kev_url = (
+            "https://www.cisa.gov/sites/default/files/feeds/known_exploited_vulnerabilities.json"
+        )
+        response = requests.get(kev_url)
+        kev_data = response.json()
+        if response.status_code != 200:
+            logger.error(
+                f"Failed to fetch the CISA Catalog of Known Exploited Vulnerabilities: {kev_url}"
+            )
+            return []
+        kev_vulnerabilities = kev_data["vulnerabilities"]
+        for kev_vul in kev_vulnerabilities:
+            alias = Alias.objects.get_or_none(alias=kev_vul["cveID"])
+            if not alias:
+                continue
+
```

Comment:
Add a `TODO` to revisit this once we have a better, improved design. The current `Improver` design is constraining.

Reply:
Done